tests/snuba/api/endpoints/test_organization_stats_v2.py

```diff
@@ -33,7 +34,9 @@ def setUp(self):
         self.project3 = self.create_project(organization=self.org2)
 
         self.user2 = self.create_user(is_superuser=False)
+        self.user3 = self.create_user(is_superuser=False)
         self.create_member(user=self.user2, organization=self.organization, role="member", teams=[])
+        self.create_member(user=self.user3, organization=self.org3, role="member", teams=[])
```

Comment:
Are these new entities still needed, or can we do the same test with the existing ones?

Reply:
I don't believe so, let me test it out

Reply:
after testing it, org3 is needed since the other orgs have projects within but i don't need user3 and could just set user2 as a member of org3.